lib/board.rb

```diff
@@ -0,0 +1,24 @@
+
+require_relative 'ui.rb' 
```

Comment:
The `Board` should have a method that checks whether it's already full.

Reply:
It is important to know if the board is full. But the way thought it was that if there are 9 moves stored by the Match, then it means that the board is full. We implemented it but in another way of thinking it.

Reply:
Done! :)